src/dynapi/SDL_dynapi.sym

```diff
@@ -0,0 +1,871 @@
+SDL_3.0.0 {
```

Comment:
Now is also the time to decide on how to name the version(s).
https://github.com/libsdl-org/SDL/pull/6613#issuecomment-1327644840 proposes:
- `libSDL3.so.0`
- `libSDL3_0`

If you want multiversioning, then some minor version should probably be added.
But as https://github.com/libsdl-org/SDL/pull/6613#issuecomment-1327724017 mentions, useless for the SDL project.

Reply:
No one uses "so" in the verdef name. Most projects use their own name, which may or may not include 'lib'. While to the linker infrastructure only uniqueness matters and you can pick arbitrary letters, uppercase seems to be popular. `SDL_3.0.0` was a fine idea.

```
libasm.so.1(ELFUTILS_1.0)(64bit)
libbsd.so.0(LIBBSD_0.0)(64bit)
libcryptsetup.so.12(CRYPTSETUP_2.0)(64bit)
libfdisk.so.1(FDISK_2.26)(64bit)
libquadmath.so.0(QUADMATH_1.0)(64bit)
libmfx-tracer.so.1(LIBMFX_1.19)(64bit)
libssl.so.3(OPENSSL_3.0.0)(64bit)
```

Reply:
I remamed it to `SDL3_0.0.0`. All uppercase seems to be the *rage*.